general_schema.json

```diff
@@ -160,11 +160,11 @@
       "description": "This MUST be a valid ORCID for the author",
       "type": "string"
     },
-    "cellannotation_setname_metadata": {
+    "labelsets": {
       "type": "array",
       "$ref": "#/definitions/cellannotation_setname_metadata"
```

Comment:
Is this part correct? There is no cellannotation_setname_metadata section in the definitions

Reply:
Could have sworn I'd already fixed that.  Thanks for spotting.  Wondering why it didn't show up on the JSON schema validation side of the checks.